src/tests/ftest/util/dmg_utils.py

```diff
@@ -358,19 +358,20 @@ def storage_replace_nvme(self, old_uuid, new_uuid, no_reint=False):
             new_uuid=new_uuid, no_reint=no_reint)
 
     def storage_query_device_health(self, uuid):
-        """Get the result of the 'dmg storage query device-health' command.
+        """Get the result of the 'dmg storage query list-devices --health' command.
 
         Args:
             uuid (str): Device UUID to query.
 
         Raises:
-            CommandFailure: if the dmg storage query device-health command fails.
+            CommandFailure: if the dmg storage query list-devices --health command fails.
 
         Returns:
             dict: the dmg json command output converted to a python dictionary
 
         """
-        return self._get_json_result(("storage", "query", "device-health"), uuid=uuid)
+        return self._get_json_result(
+            ("storage", "query", "list-device", "--health"), uuid=uuid)
```

Comment:
I suspect this is going to fail and should be
```suggestion
        return self._get_json_result(
            ("storage", "query", "list-device"), uuid=uuid, health=True)
```

Also, FYI that it uses the short option `-b` (which you can change if you want)
https://github.com/daos-stack/daos/blob/d4fbc14d59a1ab1f586dd76933b4bec5196fa30e/src/tests/ftest/util/dmg_utils_base.py#L761

Reply:
Is this actually working in CI?